roles/sushy_emulator/tasks/apply_resources.yml

```diff
@@ -0,0 +1,41 @@
+- name: Create the sushy-emulator namespace
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    name: "{{ cifmw_sushy_emulator_namespace }}"
+    kind: Namespace
+    state: present
+
+- name: Apply Sushy Emulator resources
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    name: "{{ cifmw_sushy_emulator_namespace }}"
```

Comment:
Is the field here supposed to be `namespace` or we want to name all the resources like the sushy namespace?

Reply:
actually, both the name and namespace are set in the resources templates, so maybe we can remove this field?